code-challenges/challenges-05.test.js

```diff
@@ -0,0 +1,357 @@
+'use strict';
+
```

Comment:
Challenge 3, 5 and 6 aren't passing, once you have a chance to fix them I will regrade for you!